modules/statics/src/coins.ts

```diff
@@ -1985,6 +1985,14 @@ export const coins = CoinMap.fromCoins([
     CoinKind.CRYPTO
   ),
   tofc('b364799a-e6d1-4d84-afc9-588594e850f7', 'ofctton', 'Test Ton', 9, UnderlyingAsset.TON, CoinKind.CRYPTO),
+  erc20(
+    '149c6c26-1c25-44b5-90a8-1b6b606f6604',
+    'eth:audu',
+    'Ubiquity Stablecoin AUD',
+    18,
+    '0xb749e8920b25430bd070fe859ddc84b1c99aab87',
+    UnderlyingAsset["eth:audu"]
```

Comment:
```suggestion
    UnderlyingAsset['eth:audu']
```

Reply:
you can run command  `prettier --write .` to avoid such build failure.

Reply:
Thanks for suggestion, will do.